students/ghassan/lesson06/test_new_mailroom.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python3
+
+import unittest
+from new_mailroom import main_menu, send_thankyou, donors
+
+
+class TestMailroom(unittest.TestCase):
+    def test_main_menu(self):
+        ''' this tests if the function is actually
+        returning the user's input '''
+        users_choice = main_menu()
+        self.assertEqual(users_choice, users_choice)
+
+    def test_send_thankyou_new_donor(self):
```

Comment:
The name of this function should give you a clue that your `send_thankyou` function could use some refactoring. Since each unit test is supposed to test only one thing, it's important that you try to have each your functions do only one thing. So you might look at factoring out the 'add donor' code into its own function that can be called by `send_thankyou`. This would make each separate function easier to test on their own, and make it easier to identify any potential bugs that might arise.